common/python/connectors/direct/tcs_listener/tcs_work_order_handler.py

```diff
@@ -218,11 +222,30 @@ def WorkOrderSubmit(self, **params):
             # Add to the internal FIFO
             self.workorder_list.append(wo_id)
             self.workorder_count += 1
+            # ZeroMQ for sync workorder processing
+            logger.info('TCF Workorder handler Connecting to enclave manager')
```

Comment:
I assume these are debug logs used during development. Please remove logs that are not required

Reply:
There is also `logger.debug` to provide more detailed debug level logging.